applications/asset_tracker_v2/src/events/common_module_event.h

```diff
@@ -22,12 +22,12 @@
 			  ENCODE("type"),				\
 			  profile_func)
 
-#define COMMON_EVENT_TYPE_DEFINE(ename, init_log_en, log_fn, ev_info_struct) \
+#define COMMON_EVENT_TYPE_DEFINE(ename, log_fn, ev_info_struct, flags)	\
```

Comment:
btw. Maybe we could suggest to maintainers that this seem outdated approach which is no longer needed ?